packages/core/src/m3/builtins.ts

```diff
@@ -95,26 +98,41 @@ const serializeBuiltin = (value: BuiltinPrimitive): string => {
     throw new Error(`can't serialize value of built-in primitive type: ${value}`)
 }
 
+export class SimplePrimitiveTypeDeserializer implements PrimitiveTypeDeserializer {
```

Comment:
Maybe `DefaultPrimitiveTypeDeserializer`? It's not particularly simple.

Reply:
Done!